samples/csharp/events/Program.cs

```diff
@@ -62,13 +62,13 @@ internal SearchDirectoryArgs(string dir, int totalDirs, int completedDirs)
     }
     public class FileSearcher
     {
-        public event EventHandler<FileFoundArgs> OnFoundFile;
-        internal event EventHandler<SearchDirectoryArgs> OnChangeDirectory
+        public event EventHandler<FileFoundArgs> FoundFile;
```

Comment:
I looked, and didn't find any statement either way in the Framework Design Guidelines, but if there is a convention, I'll follow it.